src/briefcase/platforms/iOS/xcode.py

```diff
@@ -351,6 +351,15 @@ def run_app(self, app: BaseConfig, udid=None, **kwargs):
             ) from e
 
         # Start log stream for the app.
+        # The following sets up a log stream filter that looks for:
+        #  1. a log sender that matches that app binary; or,
+        #  2. a log sender of that is a Python exception module,
```

Comment:
Should this be "extension module"?

Reply:
... No, I've invented a whole new type of module. It gets around all that tedious code writing and just raises the inevitable exception for you :-)

(oops - fixed :-)